docs/overview/development.md

```diff
@@ -18,6 +19,12 @@ To quickly run the tests install the local dependencies by running:
 pip install -r requirements-dev.txt
 ```
 
+You also have to install this local django-components version, initially and after every change you want to test:
```

Comment:
You can install in editable mode with `pip install -e .` or not? That should save you from running the install command again and again.

Reply:
Yup! I do that all the time

Reply:
I added the `-e` and a note on running only linters in https://github.com/django-components/django-components/pull/938